otelcollector/configmapparser/prometheus-config-merger-with-operator.rb

```diff
@@ -352,6 +357,68 @@ def populateDefaultPrometheusConfig
       end
     end
 
+    if !ENV["AZMON_PROMETHEUS_NETWORKOBSERVABILITYRETINA_SCRAPING_ENABLED"].nil? && ENV["AZMON_PROMETHEUS_NETWORKOBSERVABILITYRETINA_SCRAPING_ENABLED"].downcase == "true"
+      networkobservabilityRetinaMetricsKeepListRegex = @regexHash["NETWORKOBSERVABILITYRETINA_METRICS_KEEP_LIST_REGEX"]
+      networkobservabilityRetinaScrapeInterval = @intervalHash["NETWORKOBSERVABILITYRETINA_SCRAPE_INTERVAL"]
+      if currentControllerType == @replicasetControllerType
```

Comment:
this needs to be a no op for cfg reader sidecar as well, something like this -
if (isConfigReaderSidecar || currentControllerType == @replicasetControllerType)